test/unit/org/orgTest.ts

```diff
@@ -1043,72 +885,380 @@ describe('Org Tests', () => {
     });
   });
 
-  describe('sandboxStatus', () => {
-    let prod: Org;
-    let queryStub: SinonStub;
-    let pollStatusAndAuthStub: SinonStub;
-    const sandboxNameIn = 'test-sandbox';
-    const queryStr = `SELECT Id, Status, SandboxName, SandboxInfoId, LicenseType, CreatedDate, CopyProgress, SandboxOrganization, SourceId, Description, EndDate FROM SandboxProcess WHERE SandboxName='${sandboxNameIn}' AND Status != 'D' ORDER BY CreatedDate DESC LIMIT 1`;
-
+  describe('Sandboxes', () => {
     const statusResult = {
       records: [
         {
-          Id: '00D1u000001QQZz',
-          Status: 'Active',
+          Id: '0GR1Q000000LVf8WAG',
           SandboxName: 'test-sandbox',
-          SandboxInfoId: '00D1u000001QQZz',
-          LicenseType: 'Developer',
-          CreatedDate: '2022-01-01',
+          Status: 'Pending',
+          LicenseType: 'DEVELOPER',
+          SandboxInfoId: '0GQB0000000TVobOAG',
+          CreatedDate: '2023-09-27T20:50:26.000+0000',
+          SandboxOrg: '00D1u000001QQZz',
         },
       ],
     };
+
+    let prodTestData: MockTestOrgData;
+    let prod: Org;
+
     beforeEach(async () => {
-      const prodTestData = new MockTestOrgData();
+      prodTestData = new MockTestOrgData();
       prod = await createOrgViaAuthInfo(prodTestData.username);
-      queryStub = stubMethod($$.SANDBOX, prod.getConnection().tooling, 'query').resolves(statusResult);
-      pollStatusAndAuthStub = stubMethod($$.SANDBOX, prod, 'pollStatusAndAuth').resolves(statusResult.records[0]);
     });
 
-    it('should return sandbox status', async () => {
-      const result = await prod.sandboxStatus(sandboxNameIn, { wait: Duration.minutes(10) });
-      expect(queryStub.calledOnce).to.be.true;
-      expect(queryStub.firstCall.firstArg).to.be.equal(queryStr);
-      expect(pollStatusAndAuthStub.calledOnce).to.be.true;
-      expect(result).to.be.equal(statusResult.records[0]);
+    describe('createSandbox', () => {
```

Comment:
Most of the changes here are reorganization of the existing tests to make more sense.  I added a bunch of new tests as well in the `resumeSandbox` and `querySandboxProcess` sections.